test/controllers/ingredient_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class IngredientControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Do we plan on using minitest for this app or Rspec?

Reply:
Going to add rspec for test suite~